usagov/vote-gov-drupal
# Vote-1179 -> Scss Error Fix

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-1179](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-1179&assignee=611272e0eb7aef00694b8a97)
## Description (optional)
This PR will fix linting errors 

## Deployment and testing (required, if applicable)
### Testing steps
1. Pull down branch
2. cd into `web/themes/custom/vote_gov `
3. run `npm run dev`
4. will have to scroll up in your terminal but verify that there are no lining errors showing
